.github/workflows/downstream.yml

```diff
@@ -0,0 +1,50 @@
+name: IntegrationTest
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}
+    runs-on: ${{ matrix.os }}
+    env:
```

Comment:
This should make soft fail only. I'm not sure what's the config that would allow it.

We must _never_ get CI failures in master branch/tags that would depend on contents of other repositories.

Reply:
cc @laurentheirendt any ideas about how to conf it? :]

Reply:
what constitutes a "CI failure". I'm fine w merges if the PR passes here but not on the downstream check. CI will still show that as red x. 

@ChrisRackauckas are there settings to avoid it causing the red check? I kind of doubt it

Reply:
We might run this in PRs only to provide warning (where it should get green); and not in master to avoid breakage. Is that okay? All code is required to go through PRs anyway.

Reply:
you can always set it as an allowed failure.

Reply:
okay I think that should do it